src/generated/resources/data/modern_industrialization/recipes/compat/ae2/dyes/black/craft/me_wire_1.json

```diff
@@ -0,0 +1,26 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "fabric:load_conditions": [
```

Comment:
Sadly this is not enough if the compat is disabled... But good enough until we can use FabricMC/fabric#2548.